parsers/markdown.c

```diff
@@ -259,6 +260,93 @@ static void notifyEndOfCodeBlock (markdownSubparser *m)
 	}
 }
 
+typedef enum {
+	HT_SPACE_FOUND,
```

Comment:
Could you use HTAG_ as a prefix?
"HT_" makes me remember HT_ macros defined in main/htable.h.